lib/get-downloader.js

```diff
@@ -123,7 +122,7 @@ const getDownloader = ({
         await ensureDir(path.dirname(remoteStatus.file));
         let data;
         try {
-          data = await getData(itemsById[remoteStatus.id], api);
+          data = options.getData ? await options.getData(itemsById[remoteStatus.id], api) : await getData(itemsById[remoteStatus.id], api);
```

Comment:
it maybe time that we start using streams instead of loading all the data into memory and then putting it in a file.